Brokerages/Oanda/OandaRestApiV20.cs

```diff
@@ -187,17 +187,17 @@ public override bool PlaceOrder(Order order)
 
                     if (fill.TradeOpened != null && fill.TradeOpened.TradeID.Length > 0)
                     {
-                        marketOrderFillQuantity = fill.TradeOpened.Units.ConvertInvariant<int>();
+                        marketOrderFillQuantity = fill.TradeOpened.Units.ConvertInvariant<decimal>();
                     }
 
                     if (fill.TradeReduced != null && fill.TradeReduced.TradeID.Length > 0)
                     {
-                        marketOrderFillQuantity = fill.TradeReduced.Units.ConvertInvariant<int>();
+                        marketOrderFillQuantity = fill.TradeReduced.Units.ConvertInvariant<decimal>();
                     }
 
                     if (fill.TradesClosed != null && fill.TradesClosed.Count > 0)
                     {
-                        marketOrderFillQuantity += fill.TradesClosed.Sum(trade => trade.Units.ConvertInvariant<int>());
+                        marketOrderFillQuantity += fill.TradesClosed.Sum(trade => trade.Units.ConvertInvariant<decimal>());
                     }
 
                     marketOrderRemainingQuantity = Convert.ToInt32(order.AbsoluteQuantity - Math.Abs(marketOrderFillQuantity));
```

Comment:
`Convert.ToInt32` also needs a change? Suggest searching for `int` in the entire Oanda folder 👍